qiskit_dynamics/dispatch/__init__.py

```diff
@@ -10,10 +12,189 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Array with numpy compatibility and multiple backend support.
+"""
+=================================================
+Dispatch Module (:mod:`qiskit_dynamics.dispatch`)
+=================================================
+
+.. currentmodule:: qiskit_dynamics.dispatch
+
```

Comment:
Should this first sentence just be:

"This module contains an :class:`Array` class that wraps N-dimensional array
objects from different libraries for use with Numpy functions through a common
interface."

Not sure how to interpret the rest.